client/site-logs/components/logs-header/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { SegmentedControl } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import InlineSupportLink from 'calypso/components/inline-support-link';
+import NavigationHeader from 'calypso/components/navigation-header';
+import { navigate } from 'calypso/lib/navigate';
+
+import './style.scss';
+
+export function LogsHeader( { logType }: { logType: string } ) {
+	const options = [
+		{
+			value: 'php',
+			label: translate( 'PHP error logs' ),
+		},
+		{
+			value: 'web',
+			label: translate( 'HTTP request logs' ),
+		},
+	];
+
+	const switchType = ( newType: string ) => {
+		navigate( window.location.pathname.replace( /\/[^/]+$/, '/' + newType ) );
+	};
+
+	return (
+		<div className="logs-header">
+			<NavigationHeader
+				title={ translate( 'Logs' ) }
+				subtitle={ translate(
+					'View and download various server logs. {{link}}Learn more.{{/link}}',
+					{
+						components: {
+							link: <InlineSupportLink supportContext="site-monitoring-logs" showIcon={ false } />,
+						},
+					}
+				) }
+			/>
+			<div className="logs-header__selector-container">
+				<div className="logs-header__selector-heading">{ translate( 'Log type' ) }</div>
```

Comment:
```suggestion
				<div className="logs-header__selector-heading">{ translate( 'Log Type' ) }</div>
```

To be consistent with:
<img width="273" alt="Screenshot 2567-05-31 at 15 12 30" src="https://github.com/Automattic/wp-calypso/assets/1881481/c65ac362-152d-417a-99d1-9fc4004a8de1">

Reply:
Thanks for noticing. See the latest comment 😄